examples/arp_response.rs

```diff
@@ -168,8 +165,8 @@ fn is_arp_req(packet: &pv::Packet) -> bool {
     }
 }
 
-fn make_arp_response_packet(src_mac_addr: &MacAddr, packet: &mut pv::Packet) -> Result<(), String> {
-    let mut buffer: Vec<u8> = packet.get_buffer();
+fn make_arp_response_packet(src_mac_addr: &MacAddr, packet: &mut pv::Packet) {
+    let mut buffer: &mut [u8] = packet.get_buffer();
```

Comment:
Explicit type definition is not necessary on this kind of operation

Reply:
Changed it